app/styles/components/c-the-project.scss

```diff
@@ -0,0 +1,132 @@
+@import '../application';
+
+.c-the-project {
+  -webkit-font-smoothing: antialiased;
+
+  li {
+    color: $color-9;
+    font-family: $font-family-2;
+    font-size: $font-size-normal;
+    line-height: 1.6;
+  }
+
+  .container-the-project {
+    padding: 85px 0 90px;
+  }
+
+  .list-info {
+    list-style: none;
+
+    li {
+      padding-left: 30px;
+      position: relative;
+
+      div {
+        padding-left: 60px;
+        padding-right: 30px;
+      }
+
+      &:first-child {
+        margin-top: 15px;
+      }
+
+      & + li {
+        margin-top: 15px;
+       }
+
+      &::after {
+        background: $color-19;
+        content: '';
+        display: block;
+        height: 7px;
+        position: absolute;
+        top: 8px;
+        vertical-align: middle;
+        width: 7px;
+      }
```

Comment:
If we are going to use these styles for lists, we should unify them across the site. Can you talk with Dani about this? Thanks.